content/essentials/understanding-jsx.mdx

```diff
@@ -0,0 +1,68 @@
+import { Aside } from "~/components/configurable/Aside";
+
+# Understanding JSX
+
+[﻿JSX](https://facebook.github.io/jsx/), or JavaScript XML, is a syntax extension for JavaScript. 
+It allows you to write HTML-like code inside your JavaScript file which keeps your rendering logic and content in the same place. 
+This provides a concise and readable way to create and represent components.
+
+## How Solid uses JSX
+Solid was designed to align closely with HTML standards. It offers a distinct advantage: copy/paste solutions from resources like Stack Overflow, and direct usage of templates from design tools.
+
+What sets Solid's apart is the ability to return JSX. 
+This JSX can then be imported into other components, promoting reusability, modularity, and ease of maintenance throughout your application.
+
+Additionally, Solid's [reactive](/essentials/intro-to-reactivity) system introduces [fine-grained reactivity](/advanced/understanding-fine-grained-reactivity) with JSX.
+This ensures that only the necessary portions of the DOM are updated when changes occur in the underlying state, optimizing efficiency and overall performance.
+
+### Integrating JavaScript expressions
+In JSX, you can dynamically embed JavaScript directly into your markup using curly braces `{ }`.
+
+```jsx
+const animal = { breed: "cat", name: "Midnight" };
+const element = <p>I have a {animal.breed} named {animal.name}!</p>;
+```
+
+This allows for the injection of dynamic content into the rendered output based on the application's state or logic.
+
+### Structuring elements
+
+Unlike HTML, where you can have disconnected tags at the top level, JSX requires that a component's render method returns a single root element.
+
+<Aside>
+Note: If you are looking to return multiple elements without adding an extra node to the DOM, you would need to use a fragment (`<> ... </>`).
+
+```jsx
+<>
+  <h1>Title</h1>
+  <p>Some text.</p>
+</>
+```
+</Aside>
+
+With its structured approach, JSX maintains the familiar nested, tree-like structure found in HTML. 
+Consequently, parent-child relationships between elements are intuitive and easy to understand.
+
+To learn more about building components using JSX in Solid, visit the [components & props](/essentials/components-and-props) page.
```

Comment:
I think the learn more and the how to paragraphs can be put in Aside components, since they're kinda like hints